include/flox/resolver/manifest.hh

```diff
@@ -358,6 +94,11 @@ public:
       }
   }
 
+  // TODO: remove `manifestPath'
+  explicit GlobalManifest( GlobalManifestRaw raw )
+    : GlobalManifest( "manifest.json", std::move( raw ) )
```

Comment:
Question: Where is this `manifest.json` coming from?

Reply:
That's a temporary phony name, I'm removing it in a follow up PR.
( I had intended to remove it in this one, but want to unblock you  on `--ga-manifest` ASAP ).